index.js

```diff
@@ -1,6 +1,11 @@
 'use strict'; // eslint-disable-line strict
+const utapiServer = require('./lib/server');
+const config = require('./lib/Config');
+
+utapiServer(config);
```

Comment:
Since Utapi is included as a lib for S3, starting the server on index.js is not a good idea. You can use another file start_server.js or something instead.